src/actions/auth.js

```diff
@@ -0,0 +1,248 @@
+    };
+};
+
+// Request Reset & Reset Password
+
+exports.requestResetRequest = () => {
+
+    return {
+        type: AuthAct.REQUEST_PASSWORD_RESET_BEGIN
+    };
+};
+
+exports.requestResetSuccess = () => {
+
+    return {
+        type: AuthAct.REQUEST_PASSWORD_RESET_SUCCESS
+    };
+};
+
+exports.requestResetFailure = (errMessage) => {
+
+    return {
+        type: AuthAct.REQUEST_PASSWORD_RESET_FAILURE,
+        payload: errMessage
+    };
+};
+
+exports.resetPasswordRequest = () => {
+
+    return {
+        type: AuthAct.RESET_PASSWORD_BEGIN
+    };
+};
+
+exports.resetPasswordSuccess = () => {
+
+    return {
+        type: AuthAct.RESET_PASSWORD_SUCCESS
+    };
+};
+
+exports.resetPasswordFailure = (errMessage) => {
+
+    return {
+        type: AuthAct.RESET_PASSWORD_FAILURE,
+        payload: errMessage
+    };
+};
+
+exports.requestPasswordReset = ({ email }) => {
+
+    return (dispatch) => {
+
+        dispatch(actions.requestResetRequest());
+
+        return WebClient.post('/users/request-reset', { email }, { responseType: 'text' })
+            .then(({ data, status }) => {
+
+                dispatch(actions.requestResetSuccess());
+                return History.push('/login');
+
+            })
+            .catch((err) => {
+
+                let errMessage = 'Unable to reset password. Please try again';
+
+                if (typeof err.response !== 'undefined') {
+                    errMessage = err.response.data.message;
+                }
+
+                dispatch(actions.requestResetFailure(errMessage));
+            })
+        ;
+    };
+};
+
+exports.resetPassword = (email, newPassword, resetToken) => {
+
+    return (dispatch) => {
+
+        dispatch(actions.resetPasswordRequest());
+
+        return WebClient.post('/users/reset-password', { email, newPassword, resetToken }, { responseType: 'text' })
+            .then(({ data, status }) => {
+
+                dispatch(actions.resetPasswordSuccess());
+                return History.push('/login');
+
+            })
+            .catch((err) => {
+
+                let errMessage = 'Unable to reset password. Please try again.';
+
+                if (typeof err.response !== 'undefined') {
+                    errMessage = err.message;
+                }
+
+                dispatch(actions.resetPasswordFailure(errMessage));
+            })
+        ;
+    };
+};
+
+// StrangeAuth
+
+internals.strangeActions = StrangeAuth.makeActions({
+    login: ({ email, password, token }) => {
+
+        let authPromise;
+        let finalToken;
+
+        if ( !!token) {
+
+            finalToken = token;
+
+            authPromise = WebClient.get('/user', {
+                headers: { authorization: `Bearer ${finalToken}` }
+            });
+        }
+
+        else {
+
+            authPromise = WebClient.post('/login', { email, password }, { responseType: 'text' })
+            .then(({ data, status }) => {
+
+                if (status !== 200) {
```

Comment:
Is this needed?  I know it's in other projects, but I think non 200's get funneled to catch by axios.

Reply:
@devinivy This is Brian's question I'm not sure about! I'll dig a little into it, but figured I'd ping you too :)

Reply:
sounds like it's not needed 👌